packages/salesforce-adapter/src/types.ts

```diff
@@ -431,12 +431,8 @@ export type ClientDeployConfig = Partial<{
   flsProfiles: string[]
 }>
 
-export enum RetryStrategyName {
-  'HttpError',
-  'HTTPOrNetworkError',
-  'NetworkError',
-}
-type RetryStrategy = keyof typeof RetryStrategyName
+export const RetryStrategyNames = ['HttpError', 'HTTPOrNetworkError', 'NetworkError'] as const
```

Comment:
```suggestion
export const RETRY_STRATEGY_NAMES = ['HttpError', 'HTTPOrNetworkError', 'NetworkError'] as const
```
consts use UPPER_CASE_FORMAT

Reply:
Done.